kpi/models/import_export_task.py

```diff
@@ -509,13 +516,19 @@ def _record_last_submission_time(self, submission_stream):
                     self.last_submission_time = timestamp
             yield submission
 
+    def _get_bool_from_data(self, field: str, compare: str = 'true') -> bool:
+        return self.data.get(field, compare).lower() == 'true'
```

Comment:
i think i made this comment before flip-flopping the base branch and merging the latest `beta` into this branch. however, i'm still interested in whether or not we can avoid this; see https://github.com/kobotoolbox/kpi/pull/3012#discussion_r587823143

Reply:
@jnm I figured out why I ended up doing this:
https://github.com/kobotoolbox/kpi/blob/45462fae59b1daa85d2d2fce4952d9c456dfa4c4/kpi/models/import_export_task.py#L501-L503
https://github.com/kobotoolbox/kpi/blob/45462fae59b1daa85d2d2fce4952d9c456dfa4c4/kpi/models/import_export_task.py#L442-L446

I've changed things so that my new features expect a bool but will handle a string if the old exports endpoint is used.

Reply:
Excellent, thanks :bow: